editor/src/components/canvas/canvas-strategies/interaction-state.ts

```diff
@@ -91,12 +91,13 @@ export function createInteractionViaMouse(
   mouseDownPoint: CanvasPoint,
   modifiers: Modifiers,
   activeControl: CanvasControlType,
+  drag?: CanvasVector,
```

Comment:
This shouldn't live in here. We have a function for updating an existing interaction, which we can use for the sake of the tests

Reply:
fixed in 32a18870